src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -1236,28 +1236,63 @@ input InputUnpreferred {
 }
 
 enum RoutingErrorCode {
-    """No transit connection was found between the origin and destination withing the operating day or the next day"""
+    """
+    No transit connection was found between the origin and destination within the operating day or
+    the next day, not even sub-optimal ones.
+    """
     NO_TRANSIT_CONNECTION
 
-    """Transit connection was found, but it was outside the search window, see metadata for the next search window"""
+    """
+    A transit connection was found, but it was outside the search window. See the metadata for a token
+    for retrieving the result outside the search window.
+    """
     NO_TRANSIT_CONNECTION_IN_SEARCH_WINDOW
 
-    """The date specified is outside the range of data currently loaded into the system"""
+    """
+    The date specified is outside the range of data currently loaded into the system as it is too
+    far into the future or the past.
+
+    The specific date range of your system is configurable but also depends on the input data
```

Comment:
What is the target audience of this schema documentation? External users of the API don't have access to configure the system.

Reply:
In this specific instance it is a QA engineer who was confused by the unclear error descriptions.

We've been quite careful about not mentioning too many implementation details - and I generally agree with that - but I think we can soften our approach a little and can mention that the error depends on configurable parameters rather than a hard limit.

If you disagreed we can come up with a suitable guideline in a dev meeting.

Reply:
That is fine but maybe we shouldn't refer it to as "your system" and make it clear that changing the configuration requires access to the server configuration.